go.mod

```diff
@@ -8,3 +8,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+go 1.13
```

Comment:
This does not need to be in this PR. Please revert.

Reply:
yeah, my IDE added this, and i committed the change by mistake :(